Gemfile

```diff
@@ -20,6 +20,7 @@ group :test do
   gem 'ruby-progressbar', '~> 1.8'
   gem 'webmock', '~> 2.3.2'
   gem 'jsonschema', '~> 2.0.2'
+  gem 'm'
```

Comment:
I explained my intention in the PR message. This is a super easy way to run a single minitest test file, or even a single test in a file. If you don't want it in there, that's fine, but it makes for a better dev experience.